pontoon/teams/views.py

```diff
@@ -37,9 +38,11 @@ def teams(request):
 def team(request, locale):
     """Team dashboard."""
     locale = get_object_or_404(Locale, code=locale)
+    tiers = Tier.objects.count() or False
```

Comment:
Should this only get the tiers that are enabled for the projects that this locale is active in?